EJB Remote tests not working with Java 11

EJB remote tests added under the following issue do not run with Java 11 :
https://github.com/OpenLiberty/open-liberty/issues/7478


The HttpServlet referenced by the @TestServlet(servlet=EJBHome2xTestServlet.class, contextRoot="EJBHome2xTestWeb") annotation imported a class that was not available to the JUnit classpath. Make sure that the missing type is present on the runtime classpath of the FAT (i.e. somewhere in autoFVT/lib/ or autoFVT/build/lib/ )

java.lang.RuntimeException: The HttpServlet referenced by the @TestServlet(servlet=EJBHome2xTestServlet.class, contextRoot="EJBHome2xTestWeb") annotation imported a class that was not available to the JUnit classpath. Make sure that the missing type is present on the runtime classpath of the FAT (i.e. somewhere in autoFVT/lib/ or autoFVT/build/lib/ )
at componenttest.annotation.processor.TestServletProcessor.getTestServletMethods(TestServletProcessor.java:104)
at componenttest.annotation.processor.TestServletProcessor.getServletTests(TestServletProcessor.java:67)
at componenttest.custom.junit.runner.FATRunner.getChildren(FATRunner.java:154)
at componenttest.custom.junit.runner.FATRunner.<init>(FATRunner.java:137)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
Caused by: java.lang.NoClassDefFoundError: org.omg.CORBA.portable.OutputStream
at java.base/java.lang.J9VMInternals.prepareClassImpl(Native Method)
at java.base/java.lang.J9VMInternals.prepare(J9VMInternals.java:347)
at java.base/java.lang.Class.getMethods(Class.java:1515)
at componenttest.annotation.processor.TestServletProcessor.getTestServletMethods(TestServletProcessor.java:102)
Caused by: java.lang.ClassNotFoundException: org.omg.CORBA.portable.OutputStream
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:753)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1032)

--- a/dev/com.ibm.ws.ejbcontainer.remote_fat/build.gradle
+++ b/dev/com.ibm.ws.ejbcontainer.remote_fat/build.gradle
@@ -24,6 +24,8 @@ configurations {
 
 dependencies {
   ejbTools 'test:com.ibm.ws.ejbcontainer.fat_tools:1.+'
+  // Dependencies needed for when running on Java 9 (EE-type APIs were removed from JDK)
+  requiredLibs 'com.ibm.ws.org.apache.yoko:yoko-spec-corba:1.5.+'
 }
 
 task addEJBTools {

